src/main/resources/application.properties

```diff
@@ -10,8 +10,10 @@ spring.jpa.hibernate.ddl-auto=create-drop
 spring.jpa.show-sql=true
 spring.jpa.database=default
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
+spring.datasource.initialization-mode=always
 # server.address=172.18.128.205
```

Comment:
Maybe remove this? I can't see a reason to use this in the future.

Reply:
should remain until we have a remote server?

Reply:
Expand database population scrip with more data and move the trigger into a config file.
like data-init = true then use script